app/service/service.py

```diff
@@ -271,3 +273,18 @@ def get_stage_info(self) -> List:
         else:
             x, y = "NONE", "NONE"
         return [x, y, self.__stage_dao.get_running()]
+
+    def draw_file_preview(self, check_box_click: bool, selected_file: str, canvas: Canvas, scale: int):
```

Comment:
I don't think that service is the best place for that method

Reply:
TODO move to canvas_drawing.py